File: 0014-longest-common-prefix/0014-longest-common-prefix.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
        Arrays.sort(strs);
        String str1=strs[0];
        String str2=strs[strs.length-1];
        int e=Math.min(str1.length(),str2.length());
        String res="";
        int i=0;
        while(i<e){
            if(str1.charAt(i)==str2.charAt(i)){
                res+=str1.charAt(i);
            }
            else break;
            i++;
        }
        return res;
    }
}

File: 0014-longest-common-prefix/0014-longest-common-prefix.cs
public class Solution {
    public string LongestCommonPrefix(string[] strs) {
        if (strs == null || strs.Length == 0)
            return "";
        Array.Sort(strs);
        string s1=strs[0];
        string s2=strs[strs.Length-1];
        int i=0;
        string ans="";
        while (i < s1.Length && i < s2.Length && s1[i] == s2[i]){
            ans+=s1[i++];
        }
        return ans;
    }
}